madgraph/core/color_algebra.py

```diff
@@ -957,6 +957,8 @@ def to_canonical(self, immutable=None):
     def __eq__(self, col_str):
         """Check if two color strings are equivalent by checking if their
         canonical representations and the coefficients are equal."""
+        #misc.sprint(self.coeff, self.Nc_power, self.is_imaginary, self.to_canonical())
```

Comment:
remove these two lines?

Reply:
That one I would prefer to keep since they are helpfull for debugging